Userland/Utilities/grep.cpp

```diff
@@ -146,8 +149,16 @@ ErrorOr<int> serenity_main(Main::Arguments args)
     });
     args_parser.add_option(count_lines, "Output line count instead of line contents", "count", 'c');
     args_parser.add_positional_argument(files, "File(s) to process", "file", Core::ArgsParser::Required::No);
+    args_parser.add_option(after_context, "Print NUM lines of trailing context", "after-context", 'A', "NUM");
+    args_parser.add_option(before_context, "Print NUM lines of leading context", "before-context", 'B', "NUM");
+    args_parser.add_option(context, "Print NUM lines of output context", "context", 'C', "NUM");
     args_parser.parse(args);
 
+    if (context > 0) {
```

Comment:
not immediately obvious that those options are working in such way.

(as for me: iʼd guess that a general `context` would get overridden by more specific `before-context` and `after-context` values – either way: itʼd be nice to have this stated explictly in args and/or docs.)